packages/ui-dom-utils/src/addInputModeListener.ts

```diff
@@ -119,20 +113,19 @@ const addDownListeners = () => {
 }
 
 const removeListeners = () => {
-  // @ts-expect-error ts-migrate(7005) FIXME: Variable '_moveListeners' implicitly has an 'any[]... Remove this comment to see the full error message
   _moveListeners.forEach((listener) => listener.remove())
   _moveListeners = []
-  // @ts-expect-error ts-migrate(7005) FIXME: Variable '_downListeners' implicitly has an 'any[]... Remove this comment to see the full error message
   _downListeners.forEach((listener) => listener.remove())
   _downListeners = []
 }
 
-// @ts-expect-error ts-migrate(7031) FIXME: Binding element 'onInputModeChange' implicitly has... Remove this comment to see the full error message
-const addInputModeListener = ({ onInputModeChange }) => {
+const addInputModeListener = (handlerObject: {
+  onInputModeChange: (...args: any[]) => any
+}) => {
+  const { onInputModeChange } = handlerObject
   const id = _registeredCount++
 
   if (typeof onInputModeChange === 'function') {
```

Comment:
this is always a function, no need for this check

Reply:
As we discussed, we should keep this, because not having this check could break the `forEach` above, because it won't be "callable".